cmd/install.go

```diff
@@ -47,27 +44,19 @@ var installCommand = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		imageInspect, _, err := cli.ImageInspectWithRaw(context.Background(), imageName)
+
+		ctx := context.Background()
+
+		imageInspect, err := cli.ImageInspect(ctx, imageName)
```

Comment:
why do we need to provide the context from the top? It looks like we are not doing anything about it yet, can't it be handled directly in the client wrapper?

Reply:
I think it's just nice to have a single instance of context flow through the program as it would be nice to add cancelation via signals in the future.

Reply:
@tjamet I can revert to passing a new context to things if you prefer? In my opinion it seems more correct to maintain a singleton for a session of the cli though